Automattic/jetpack
# DIFM : Use atomic blog sticker to detect whether DIFM is in progress

Fixes  Automattic/martech#1105

#### Changes proposed in this Pull Request:

* Accessing stickers on Atomic sites are done via a different function. This change adds a fallback for Atomic sites to use blog sticker details provided in persistent data.

#### Does this pull request change what data or activity we track or use?
No

#### Other information:


#### Testing instructions:
* Sandbox public-api, dev-mc
* Apply D89083-code to your sandbox
* Create WoA site and push these plugin changes into the atomic site
* On the atomic site purchase DIFM
    * Go to `/start/do-it-for-me?flags=difm/allow-woa-sites` 
    * Select `Existing WordPress.com site`
    * In the next page make sure the feature flag `flags=difm/allow-woa-sites` is there or add it if not.
    * Select the above woa site and complete the purchase
* ~You will redirected to home~ go back to the calypso home page of the atomic site and, only the following menus should be visible.

<img width="1716" alt="image" src="https://user-images.githubusercontent.com/3422709/193906009-e804ae10-4381-40a2-b179-ea39c03e8f9b.png">

* Go to the site report card and remove the difm-lite-in-progress sticker. 
* Menu's should now be visible as usual.
